tests/global_config_gcp.py

```
@@ -0,0 +1,34 @@
+            "volume_mode": DataVolume.VolumeMode.BLOCK,
+            "access_mode": DataVolume.AccessMode.RWX,
+            "snapshot": True,
+            "online_resize": True,
+            "wffc": True,
+            "default": True,
+        }
+    },
+]
+
+for _dir in dir():
+    if not config:  # noqa: F821
```

Pylint (3.3.7)

[error] Using variable 'config' before assignment

(E0601)